project_one.html

```diff
@@ -15,195 +15,308 @@
-            <span class="side"> BACK END DEV </span>
-            <img class="dot" src="./images/Counter.png" alt="" />
-            <span class="side"> 2015 </span>
+
+        <br />
+        <br />
+        <div class="card">
+          <div class="snap">
+            <img src="./images/Snap3.png" alt="img" />
           </div>
-          <p class="para">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <div class="links">
-            <ul>
-              <li class="link">html</li>
-              <li class="link">css</li>
-              <li class="link">javaScript</li>
-            </ul>
+          <div class="websnap">
+            <img src="./Portoflio/Snapshoot3.png" alt="" />
           </div>
-          <a href="#" class="pro"> See Project </a>
-        </section>
-      </div>
+          <section class="main">
+            <h2>Tonic</h2>
+            <div class="mid">
+              <span class="can"> CANOPY </span>
+              <img class="dot" src="./images/Counter.png" alt="" />
+              <span class="side"> BACK END DEV </span>
+              <img class="dot" src="./images/Counter.png" alt="" />
+              <span class="side"> 2015 </span>
+            </div>
+            <p class="para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="links">
+              <ul>
+                <li class="link">html</li>
+                <li class="link">css</li>
+                <li class="link">javaScript</li>
+              </ul>
+            </div>
+            <a href="#" class="pro"> See Project </a>
+          </section>
+        </div>
 
+        <br />
+        <br />
+
+        <div class="card webcard">
+          <div class="snap">
+            <img src="./images/Snap4.png" alt="img" />
+          </div>
+          <div class="websnap1">
+            <img src="./Portoflio/Snapshoot4.png" alt="" />
+          </div>
+          <section class="main1">
+            <h2>Multi-Post</h2>
+            <h2>Stories</h2>
+            <div class="mid">
+              <span class="can"> CANOPY </span>
+              <img class="dot" src="./images/Counter.png" alt="" />
+              <span class="side"> BACK END DEV </span>
+              <img class="dot" src="./images/Counter.png" alt="" />
+              <span class="side"> 2015 </span>
+            </div>
+            <p class="para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="links">
+              <ul>
+                <li class="link">html</li>
+                <li class="link">css</li>
+                <li class="link">javaScript</li>
+              </ul>
+            </div>
+            <a href="#" class="pro"> See Project </a>
+          </section>
+        </div>
```

Comment:
Kindly fix the following problems with your cards:
- They are not centered properly
- Your project card images are not well-centered vertically
- Your technologies tags have too much left and right padding.
- Images of cards that have images to the right side are too far from right edges of those cards.

<img width="1280" alt="Zrzut ekranu 2022-03-26 o 15 53 42" src="https://user-images.githubusercontent.com/87897753/160245229-f418d020-b9c1-415a-8efe-c2f154b8193d.png">